doc/sphinxext/gen_rst.py

```diff
@@ -380,19 +445,26 @@ def resolve(self, cobj, this_url):
     :align: center
 """
 
+# The following dictionary contains the information used to create the
+# thumbnails for the front page of the scikit-learn home page.
```

Comment:
Shouldn't we purge references to scikit-learn (other than the acknowledgment at the top)?